src/main/java/frc/utils/Alert.java

```diff
@@ -0,0 +1,30 @@
+package frc.utils;
+
+import edu.wpi.first.wpilibj.Timer;
+import frc.robot.constants.LogPaths;
+import org.littletonrobotics.junction.Logger;
+
+public class Alert {
+
+    public enum AlertType {
+        ERROR,
+        WARNING;
+    }
+
+    private final AlertType type;
+    private final String name;
+
+    public Alert(AlertType alertType, String alertName){
```

Comment:
kinda weird that the parameters dont share a name with the vars in the class (type and name)